lib/routes/emails.js

```diff
@@ -763,13 +744,6 @@ module.exports = (log, db, mailer, config, customs, push) => {
           .then((result) => {
             account = result
```

Comment:
Should be able to remove the account lookup here and replace https://github.com/mozilla/fxa-auth-server/pull/2121/files#diff-cf65808c0d0da40a5f824db418c4ccd7L802 with the `uid` from session.